commands/inspect.go

```diff
@@ -0,0 +1,175 @@
+package commands
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+	"text/tabwriter"
+	"time"
+
+	"github.com/docker/cli/cli"
+	"github.com/docker/cli/cli/command"
+	"github.com/moby/buildkit/util/appcontext"
+	"github.com/spf13/cobra"
+	"github.com/tonistiigi/buildx/build"
+	"github.com/tonistiigi/buildx/driver"
+	"github.com/tonistiigi/buildx/store"
+	"github.com/tonistiigi/buildx/util/progress"
+	"golang.org/x/sync/errgroup"
+)
+
+type inspectOptions struct {
+	bootstrap bool
+}
+
+type dinfo struct {
+	di        *build.DriverInfo
+	info      *driver.Info
+	platforms []string
+	err       error
+}
+
+type nginfo struct {
+	ng      *store.NodeGroup
+	drivers []dinfo
+	err     error
+}
+
+func runInspect(dockerCli command.Cli, in inspectOptions, args []string) error {
+	ctx := appcontext.Context()
+
+	txn, release, err := getStore(dockerCli)
+	if err != nil {
+		return err
+	}
+	defer release()
+
+	var ng *store.NodeGroup
+
+	if len(args) > 0 {
+		ng, err = getNodeGroup(txn, dockerCli, args[0])
+		if err != nil {
+			return err
+		}
+	} else {
+		ng, err = getCurrentInstance(txn, dockerCli)
+		if err != nil {
+			return err
+		}
+	}
+
+	if ng == nil {
+		ng = &store.NodeGroup{
+			Name: "default",
+			Nodes: []store.Node{{
+				Name:     "default",
+				Endpoint: "default",
+			}},
+		}
+	}
+
+	ngi := &nginfo{ng: ng}
+
+	timeoutCtx, cancel := context.WithTimeout(ctx, 5*time.Second)
+	defer cancel()
+
+	err = loadNodeGroupData(timeoutCtx, dockerCli, ngi)
+
+	if in.bootstrap {
+		if err := boot(ctx, ngi); err != nil {
+			return err
+		}
+		ngi = &nginfo{ng: ng}
```

Comment:
dupe

Reply:
no, we need to reload from clean state after boot